test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +42,41 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+
+    // this is a method for testing the method Utils.isAnyNull
+    @Test 
+    public void isAnyNullTest() {
+        // empty list --> why does it lead to test failure
+        //assertIsAnyNull();
+        //assertAllNotNull();
+
+        // only one object
+        assertAllNotNull("");
+        assertAllNotNull(Integer.valueOf("2"));
+        assertAllNotNull(1);
+        assertIsAnyNull((Object) null);
+
+        // three objects, containing at least 1 null object, at different positions
+        assertIsAnyNull((Object) null, "j", "p");
+        assertIsAnyNull("j", (Object) null, "p");
+        assertIsAnyNull("j", "p", (Object) null);
+        assertIsAnyNull("j", (Object) null, (Object) null);
+        assertIsAnyNull((Object) null, (Object) null, (Object) null);
+
+
+        // three none null objects
```

Comment:
Be careful with typos as comments are used for telling others what you are doing